README.md

```diff
@@ -115,6 +118,19 @@ e.g. by standardizing Kubernetes labels using an
 [Admission Webhook](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/)
 that ensures that there are no possible conflicts.
 
+#### Enabling Kubernetes labels
+
+In order to prevent high cardinality of the `*_labels` family of metrics, kube-state-metrics only exposes specifically allowed labels.
+The labels can be configured through the use of the `--labels-allow-list` flag. The flag accepts a comma separated
+list of `<metric>=[<labels>]`, where `<labels>` are again comma separated full label names. For example:
+
+```
+--labels-allow-list kube_deployment_labels[label_app,label_env,deployment,namespace],kube_pod_labels[label_app,label_env,pod,namespace]
```

Comment:
```suggestion
--labels-allow-list kube_deployment_labels=[label_app,label_env,deployment,namespace],kube_pod_labels=[label_app,label_env,pod,namespace]
```

Reply:
The namespace and pod name will be on by default, its currently a bug that is being fixed, rest lgtm.

Reply:
My review was not about namespace an pod name, but about missing equals. They are now added so it's fine

Reply:
@gturbat agreed and exactly why I submitted the last edit. But I have to admit that the comment below was throwing me off :sweat_smile:. But as I also read the mails from Github, it actually became clear what your intention was.